RELEASE-NOTE-STABLE-3.1.md

```diff
@@ -0,0 +1,1050 @@
+* b613321c switch-to-containers: do not fail when stopping the nfs-ganesha service (Andrew Schoen)
+* 0b4b60e3 switch-to-containers: do not fail when stopping the ceph-mgr daemon (Andrew Schoen)
+* 997edea2 rolling_update: do not fail the playbook if nfs-ganesha is not present (Andrew Schoen)
+* 9c5e8ce7 doc: corrected a typo (Aviolat Romain)
+* 0b55abe3 mon: always run ceph-create-keys (Sébastien Han)
+* ad54e192 rgw: disable legacy rgw service unit (Sébastien Han)
+* 895949d6 osd: fix check gpt (Guillaume Abrioux)
+* c5b7b371 purge-cluster: clean some code (Guillaume Abrioux)
+* eeedefdf purge-cluster: wipe disk using dd (Guillaume Abrioux)
+* 39f2bfd5 fix jewel scenarios on container (Sébastien Han)
+* c315f81d site-docker: ability to disable fact sharing (Sébastien Han)
+* bbc79765 osd: best effort if no device is found during activation (Sébastien Han)
+* 20078583 rolling_update: do not require root to answer question (Sébastien Han)
+* dfbef836 nfs: fix package install for debian/suss systems (Sébastien Han)
+* 50a848dc Rename fact docker_version to ceph_docker_version (Christian Berendt)
+* 162b7d2b roles: ceph-mgr: Install the ceph-mgr package on SUSE (Markos Chandras)
+* 15287c66 contrib: do not skip ci on backport (Sébastien Han)
+* a24fd1cf client: don't make `osd_pool_default_pg_num` mandatory (Guillaume Abrioux)
+* ab1dd302 client: don't try to generate keys (Guillaume Abrioux)
+* 26afe46e docker: add missing condition for selinux tasks (Guillaume Abrioux)
+* 7eaf4443 default: look for the right return code on socket stat in-use (Sébastien Han)
+* 8cba4426 Add flags for OSD 'docker run --cpuset-{cpus,mems}' (John Fulton)
+* a8a2c13f firewall: add mds, nfs, restapi and iscsi ports, remove 'configure_firewall' variable used for conditional * execution. Include the task only on rpm-based systems. (Eduard Egorov)
+* 6a5e0da3 firewall: configure firewalld if it's already installed on the host (#2192). (Eduard Egorov)
+* 73a20e9b Revert "tests: set CEPH_STABLE_RELEASE in ceph-build" (Guillaume Abrioux)
+* 5676fa23 Convert interface names to underscores for facts (Major Hayden)
+* d7dadc3e ceph-osd: respect nvme partitions when device is a disk. (Konstantin Shalygin)
+* 6a9b5c96 defaults: fix CI issue with ceph_uid fact (Guillaume Abrioux)
+* 788c3f35 ceph-osd: adds osd_objectstore to the name when using the ceph_volume module (Andrew Schoen)
+* 15db3323 tests for the ceph_volume module (Andrew Schoen)
+* 4b4680d1 refactor ceph_volume.py so it's easier to test (Andrew Schoen)
+* 5e3d8dbf ceph-osd: use the cluster param with the ceph_volume module (Andrew Schoen)
+* 7721aa37 ceph_volume: adds the cluster param (Andrew Schoen)
+* 423166f6 ceph-osd: use the new ceph_volume module for the lvm scenario (Andrew Schoen)
+* 61b4a1be adds a ceph_volume module (Andrew Schoen)
+* a9981b41 Vagrantfile: Fixed repeated OSD controller creation (Prisacari Dmitrii)
+* 0ea1811f Merge pull request #2226 from andymcc/gpt_mklabel (Sébastien Han)
```

Comment:
I don't think these merge entries are that useful, specially in the context of a very large set of changes

Reply:
Indeed, removed them, thanks!